CONTRIBUTING.md

```diff
@@ -1,60 +1,202 @@
+|                      | A change is made that does not affect the public API of the component | `patch`            |
+| Component Props      | An optional prop is added                                             | `minor`            |
+|                      | A required prop is added                                              | `major` (breaking) |
+|                      | A prop's type is expanded                                             | `minor`            |
+|                      | A prop's type is narrowed                                             | `major` (breaking) |
+|                      | A prop is deprecated                                                  | `minor`            |
+|                      | A prop is removed                                                     | `major` (breaking) |
+| Component Styles/CSS | Adding a new css selector                                             | `minor`            |
+|                      | Removing a css selector                                               | `major` (breaking) |
+|                      | Adding a css variable                                                 | `minor`            |
+|                      | Removing a css variable                                               | `major` (breaking) |
+|                      | Modifying css properties                                              | `patch` or `minor` |
+
+### Pull Requests
+
+We aim to review pull requests within a day or two. If your changes are time sensitive, feel free to request an expedited review on GitHub or reach out in the private `#cauldron` slack channel.
+
+Once approved by a member of the Cauldron team, your pull request can be merged at any time but _must_ be squash merged.
+
+#### Previewing Changes
+
+Cauldron documentation is deployed automatically via [amplify web previews for pull requests](https://docs.aws.amazon.com/amplify/latest/userguide/pr-previews.html) with each commit to a PR. To view the preview of your changes, navigate to your PR and find the comment from the `aws-amplify` bot, which will include a link to the preview site for that PR. The preview site will only persist for as long as the PR remains opened, and will be deleted when closed.
+
+### Testing Strategies
+
+Testing methodology should account for both testing the interface of a component as input/output testing and with how an end-user would interact with the component. Ideally, implementation details are avoided unless necessary. Examples below of things to consider when writing tests for a component:
+
+- Including all variants in tests (i.e. default, primary, secondary)
+- Including different interactive states (i.e. focus, expanded, disabled)
+- Including testing different behaviors for functionality related to different layouts (i.e. i.e. responsiveness, media queries, overflow)
+- Component api and events (i.e. callbacks, click, keyboard events)
```

Comment:
```suggestion
- Component API and events (i.e. callbacks, clicks, keyboard events)
```

Reply:
This was supposed to be referencing click / keyboard events, so perhaps that's what it should be changed to?

Reply:
So "i.e. callbacks, click / keyboard events"?

Reply:
Correct, made the change!